packages/nuka/src/default-controls.tsx

```diff
@@ -190,22 +185,12 @@ export const PagingDots = (props: ControlProps) => {
     []
   );
 
-  const indexes = useMemo(
```

Comment:
The identity of the array here wasn't important, nor is `getDotIndexes` particularly costly, so I took out the memoization.